### ISSUE TITLE
Filtering a category's transactions doesn't update the total

### ISSUE THREAD

**Issue by Deguello** [actualbudget/releases#244]
1. Click the Spent column on a category.

2. The sum of the transactions in the resulting register view matches the category total, as expected.

3. Apply a filter that excludes one or more of the listed transactions.

4. The transactions which don't match the filter disappear, as expected.

5. The category total at the top left of the screen doesn't update to reflect the actual total of the remaining transactions (it still reflects the category total).

I expect the displayed total to update to reflect the sum of the displayed transactions rather than staying the same as the category total.

**Comment by Kidglove57** [comment]
Yes, I had noticed something similar when selecting transactions and found that I always had to manually deselect everything and then start from scratch. Which is not ideal.
